src/components/List/index.js

```diff
@@ -19,7 +20,7 @@ export default function List (props) {
   typeConvert[offerString] = 'offers';
   typeConvert[requestString] = 'needs';
 
-  const [category, setCategory] = useState('All');
+  const [category, setCategory] = useState('all');
   const [data, setData] = useState(null);
   useEffect(() => {
```

Comment:
Unrelated to this PR: You could merge this pair of `useState` and `useEffect` into it's own hook together named something like `useData`.